packages/augur-ui/src/modules/market/containers/trading-form.ts

```diff
@@ -20,17 +20,30 @@ import {
   IS_CONNECTION_TRAY_OPEN
 } from "modules/auth/actions/auth-status";
 import { selectSortedMarketOutcomes } from "modules/markets/selectors/market";
-
+import orderAndAssignCumulativeShares from "modules/markets/helpers/order-and-assign-cumulative-shares";
+import { formatOrderBook } from 'modules/create-market/helpers/format-order-book';
 
 const mapStateToProps = (state, ownProps) => {
   const { authStatus, loginAccount } = state;
 
-  const hasFunds = !!state.loginAccount.balances.eth && !!state.loginAccount.balances.dai
+  const hasFunds = !!state.loginAccount.balances.eth && !!state.loginAccount.balances.dai;
+
+  const selectedOutcomeId = (ownProps.selectedOutcomeId !== undefined && ownProps.selectedOutcomeId !== null) ? ownProps.selectedOutcomeId : market.defaultSelectedOutcomeId;
+  let outcomeOrderBook = ownProps.market.orderBook[selectedOutcomeId];
```

Comment:
orderbook isn't on the market, use the order book selector, select-order-book.ts